src/test/java/core/basesyntax/service/FileReaderServiceTest.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceTest {
+    private static FileReaderService fileReaderService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void pathToFile_NullNotOk() {
```

Comment:
```suggestion
    public void readFromFile_NullPath_NotOk() {
```

dont forget about method naming (There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: <methodUnderTest>_<state>_<expectedBehavior>; For example, if we are testing the method register with a null
user's age the test method name should be register_nullAge_notOk. notOk is because the test expects the register method to throw an exception.)
fix an all places